modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -86,7 +89,8 @@ def veteran_meta
         @evss_claim[:veteran] ||= {}
         # EVSS Docker needs currentlyVAEmployee, 526 schema uses currentVaEmployee
         @evss_claim[:veteran][:currentlyVAEmployee] = @data.dig(:veteranIdentification, :currentVaEmployee)
-        @evss_claim[:veteran][:emailAddress] = @data.dig(:veteranIdentification, :emailAddress, :email)
+        email_address = @data.dig(:veteranIdentification, :emailAddress, :email)
+        @evss_claim[:veteran][:emailAddress] = email_address unless email_address.nil?
```

Comment:
before, this would trigger a validation error in EVSS Docker as we sent the email of `nil`, so avoiding sending any `emailAddress` allowed a submission.